src/Hl7.Fhir.Support.Poco.Tests/NewPocoSerializers/SummaryFilterTests.cs

```diff
@@ -16,7 +16,7 @@ public void BundleFilter()
             var epf = new ElementPrefixFilter("elem");
             var bf = new BundleFilter(epf);
 
-            var inspector = ModelInspector.ForAssembly(typeof(TestBundle).Assembly);
+            var inspector = ModelInspector.ForAssembly(typeof(TestPatient).Assembly);
 
             bf.EnterObject(null, inspector.FindClassMapping("Bundle"));
```

Comment:
The reason this was typeof(TestBundle), is that we navigate here into a Bundle. Maybe this should be `ForAssembly(typeof(Bundle))` now?